docs/howto_guides/api_gui.md

```diff
@@ -0,0 +1,161 @@
+<br/>
+
+![find get activities by activity id](../_static/images/howto_guides/api_gui/find_get_activities_activity_id.png)
+If the activity identifier is known, the activity metadata can be retrieved using the `GET /activities/activity_id` endpoint. Note that only one metadata record for an activity may be returned at a time using this method.<br/>
+<br/>
+
+For more information and to see more examples of __find__ endpoints outside of the [autogenerated user interface](https://api.microbiomedata.org/docs#/find), please visit: https://api.microbiomedata.org/search<br/>
+<br/>
+
+#### Find Endpoint Example: get all studies that have EMSL (Environmental Molecular Sciences Laboratory) related funding
+
+1. Click on the drop down arrow to the right side of the **`GET /studies`** endpoint
+![find example step 1](../_static/images/howto_guides/api_gui/find_example_step1.png)
+2. Click **Try it out** in the upper right of the expanded endpoint box
+![find example step 2](../_static/images/howto_guides/api_gui/find_example_step2.png)
+3. Enter in parameters. In this case, we will input `funding_sources.search:EMSL` into the **filter** parameter. The `.search` performs a full text search to find studies with `funding_sources` that have the word "EMSL" in its value.
+4. Click **Execute**
+![find example step 3 and step 4](../_static/images/howto_guides/api_gui/find_example_step3and4.png)
+5. View the results in JSON format, available to download by clicking **Download**, or copy the results by clicking the clipboard icon in the bottom right corner of the response. In this case, two studies were retrieved.
+![find example step 5](../_static/images/howto_guides/api_gui/find_example_step5.png)<br/>
+<br/>
+
+- Note that a curl request and request URL are provided as well for command line usage:
+![find example note](../_static/images/howto_guides/api_gui/find_example_note.png)
+
+#### ___Metadata___ Endpoints
+
+The [metadata endpoints](https://api.microbiomedata.org/docs#/metadata) can be used to get and filter metadata from collection set types (including studies, biosamples, activites, and data objects as discussed in the __find__ section), as well as validate and submit updates to existing metadata to the data portal as a spreadsheet or json file. 
+
+The syntax for the filter parameter of the __metadata__ endpoints is slightly different than that of the __find__ endpoints using [MongoDB-like language querying](https://www.mongodb.com/docs/manual/tutorial/query-documents/) instead of the compact syntax the __find__ endpoints use. The applicable parameters of the __metadata__ endpoints, with acceptable syntax and examples, are in the table below.
```

Comment:
this is a little hard to read... recommend 

Unlike the compact syntax used in the __find__ endpoints, the syntax for the filter parameter of the __metadata__ endpoints use [MongoDB-like language querying](https://www.mongodb.com/docs/manual/tutorial/query-documents/). The applicable parameters of the __metadata__ endpoints, with acceptable syntax and examples, are in the table below.

Reply:
Thanks! I replaced this paragraph with your suggestion [8ec86c6](https://github.com/microbiomedata/NMDC_documentation/pull/27/commits/8ec86c6cdde32a6d109ee0300067b2d2ebc3e89b)